webrtc/udp_transport.nim

```diff
@@ -13,23 +13,23 @@ import chronos, chronicles
 logScope:
   topics = "webrtc udp"
 
-# UdpConn is a small wrapper of the chronos DatagramTransport.
+# UdpTransport is a small wrapper of the chronos DatagramTransport.
 # It's the simplest solution we found to store the message and
-# the remote address used by the underlying protocols (dtls/sctp etc...)
+# the remote address used by the wrapped protocols (dtls/sctp...)
 
 type
   UdpPacketInfo* = tuple
     message: seq[byte]
     raddr: TransportAddress
 
-  UdpConn* = ref object
+  UdpTransport* = ref object
```

Comment:
No, it'll not be directly used by nim-libp2p. I change it because an UDP Connection is nonsensical. There's no connection in UDP and this code behaves more like a transport than a connection.